soc/common_soc.mk

```diff
@@ -35,29 +35,34 @@ endif
 PROJ_DIR:=  $(CFU_ROOT)/proj/$(PROJ)
 CFU_V:=     $(PROJ_DIR)/cfu.v
 CFU_ARGS:=  --cpu-cfu $(CFU_V)
+TARGET_ARGS:= --target $(TARGET)
 
-SOC_NAME:=  nexys_video.$(PROJ)
+SOC_NAME:=  common_soc.$(PROJ)
```

Comment:
Can we change this to `SOC_NAME:= $(TARGET).$(PROJ)` ?

Reply:
Sure, I changed that but to make it work I had to add [a statement](https://github.com/google/CFU-Playground/pull/105/files#diff-13f94e0336e4f0044b0809865aa44e3d111cb67a44b948d2a98f0268984eb5f6R63) that makes sure `$(TARGET)` flag is properly set. It's necessary for platforms other than `common_soc`